src/ucp/api/ucp.h

```diff
@@ -174,10 +174,13 @@ enum ucp_dt_type {
  * @ingroup UCP_MEM
  * @brief UCP memory mapping flags.
  *
- * The enumeration list describes the datatypes supported by UCP.
+ * The enumeration list describes the memory mapping flags supported by @ref
+ * ucp_mem_map() function.
  */
 enum {
-    UCP_MEM_MAP_NONBLOCK = UCS_BIT(0) /* Avoid populating the pages in the mapping up-front */
+    UCP_MEM_MAP_NONBLOCK = UCS_BIT(0) /*< Complete the mapping faster, possible by
```

Comment:
I would suggest elaborate even more. The actual mapping can be delayed to the communication request (or something like this).